src/Features/Core/Portable/ConvertAnonymousTypeToClass/AbstractConvertAnonymousTypeToClassCodeRefactoringProvider.cs

```diff
@@ -72,8 +72,7 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             // it matches even when caret is next to some tokens within the anonymous object creation node.
             // E.g.: `var a = new [||]{ b=1,[||] c=2 };` both match due to the caret being next to `,` and `{`.
             var helper = document.GetLanguageService<IRefactoringHelpersService>();
-            var anonymousObject = await helper.TryGetSelectedNodeAsync<TAnonymousObjectCreationExpressionSyntax>(
-                document, span, cancellationToken).ConfigureAwait(false);
+            var anonymousObject = await document.TryGetSelectedNodeAsync<TAnonymousObjectCreationExpressionSyntax>(span, cancellationToken).ConfigureAwait(false);
```

Comment:
fwiw, i like the rule of: should be visible on github.com at 1080p (most common resolution for VS devs).  That works out to about 120col.